src/main/scala/metricsLambdas/logic/CapiAPILogic.scala

```diff
@@ -12,11 +12,12 @@ import io.circe.syntax._
 import metricsLambdas.Config._
 import metricsLambdas.{KinesisWriter, Logging}
 import org.joda.time.{DateTime, DateTimeZone}
+import java.time.Instant
```

Comment:
Any reason for the switch to `Instant`? I'm not against it I'm just curious

Reply:
We needed to update the capi scala client to be able to query commissioning length and the new version requires that dates should be passed as instants.